spec/features/pets/user_can_see_all_pets_at_a_shelter_spec.rb

```diff
@@ -27,6 +27,7 @@
     expect(page).to have_content(@pet2.name)
     expect(page).to have_content(@pet2.approximate_age)
     expect(page).to have_content(@pet2.sex)
+    expect(page).to have_content(@shelter.pet_count)
```

Comment:
This gives me a false positive because there is the same number somewhere else on the page. 
I may try to switch this to a css capybara test.